### ISSUE TITLE
Add to website links "In the wild"

### ISSUE THREAD

**Issue by onlykey** [keybase/kbpgp#162]
On your web page you list examples of devices that use kbpgp.js and mention to hit you up on Github to add content. We have launched a Kickstarter ([OnlyKey Quantum](https://www.kickstarter.com/projects/1048259057/onlykey-quantum-future-ready-encryption-for-everyo)) where we are building a hardware device that uses kbpgp.js. Check it out and it would be great if you added a link to this.

**Comment by wswoodruff** [comment]
@onlykey Question from your site (crp.to): Would you guys consider `kbpgp.js` military-grade encryption? Or is GPG / PGP itself, military-grade encryption?

**Comment by onlykey** [comment]
@wswoodruff Military grade encryption is a marketing term, saying things like AES-256-GCM or RSA-4096 means nothing to most general users.

**Comment by wswoodruff** [comment]
Have you guys played with elliptical curve encryption yet?

**Comment by onlykey** [comment]
@wswoodruff Yes, up to 32 ECC keys are supported of types X25519, NIST P256, and secp256k1.

**Comment by wswoodruff** [comment]
Thanks for the info your project is so cool! I'm planning on using `kbpgp` on an open-source project and would LOVE to have this `"In the wild"` section!

**Comment by onlykey** [comment]
@wswoodruff Thanks! If you are using kbpgp on your open-source project it's probably also OnlyKey compatible with some minor changes to kbpgp. I went ahead and added this to our FAQ.

**Comment by wswoodruff** [comment]
Thank you! I will definitely look into that!
